bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolder.java

```diff
@@ -2062,6 +2062,9 @@ void onPaint(Event event) {
 	}
 
 	GC gc = event.gc;
+	if (gc == null) {
```

Comment:
It is illegal to have event.gc set to null on paint. The caller (org.eclipse.e4.ui.workbench.renderers.swt.StackRendererTest.testOnboardingIsHiddenWhenEditorOpened()) should be fixed.

Reply:
Would https://github.com/eclipse-platform/eclipse.platform.ui/pull/2369 be a correct fix for this?

Reply:
Yes.